wgengine/userspace.go

```diff
@@ -551,13 +550,6 @@ func forceFullWireguardConfig(numPeers int) bool {
 	if opt := controlclient.TrimWGConfig(); opt != "" {
 		return !opt.EqualBool(true)
 	}
-
-	// On iOS with large networks, it's critical, so turn on trimming.
-	// Otherwise we run out of memory from wireguard-go goroutine stacks+buffers.
-	// This will be the default later for all platforms and network sizes.
-	if numPeers > 50 && version.OS() == "iOS" {
-		return false
-	}
```

Comment:
Intentional change?

Reply:
I can send this separately as well. It's old and equivalent to the "return false" on the next line.

Reply:
As long as this is intentional I'm fine with it living here.

Reply:
already submitted separately with updated comments too